articles/quickstarts/qrng.md

```diff
@@ -40,24 +53,79 @@ When a `Qubit` is de-allocated it must be explicitly set back to the `Zero` stat
 
 In the Bloch sphere, the north pole represents the classical value **0** and the south pole represents the classical value **1**. Any superposition can be represented by a point on the sphere (represented by an arrow). The closer the end of the arrow to a pole the higher the probability the qubit collapses into the classical value assigned to that pole when measured. For example, the qubit state represented by the red arrow below has a higher probability of giving the value **0** if we measure it.
 
-<img src="~/media/qrng-Bloch.png" width="175" alt="A qubit state with a high probability of measuring zero">
+<img src="~/media/qrng-Bloch.png" width="175">
```

Comment:
Why was the alt-text removed here? We should make sure to preserve alt-text if possible, as that's a critical accessibility feature.